### PR TITLE
Filter the content list

### PR DESCRIPTION
**Pull request by kallehauge**
Enables the content filter list and alter the default criteria (removed language and added author).

It's not easily possible to add filtration on content since it's paragraphs referenced items. It allows to filter based on target_id etc. but out of the box it's only possible to filter on title.

### CODE REVIEW

File: web/modules/custom/dbcdk_community_moderation/dbcdk_community_moderation.module [hunk 1]
```diff
@@ -4,22 +4,6 @@
...
- */
-function dbcdk_community_moderation_user_login(User $account) {
-  // Redirect moderators to the flagged content list.
-  if (in_array('admin_moderator', $account->getRoles())) {
-    $url = new Url('page_manager.page_view_dbcdk_community_flagged_content');
-    $redirect = new RedirectResponse($url->toString());
-    $redirect->send();
-  }
-}
-
```
Comment (kasperg) [inline web/modules/custom/dbcdk_community_moderation/dbcdk_community_moderation.module:22]: Why is this included in the PR?

It does not seem related to the change at hand.

File: web/modules/custom/dbcdk_community/dbcdk_community.module [hunk 2]
```diff
@@ -4,6 +4,35 @@
...
+    array_flip($account->getRoles())
+  );
+  $route = array_shift($user_role_routes);
+  if (!empty($route)) {
+    $url = new Url($route);
+    $redirect = new RedirectResponse($url->toString());
+    $redirect->send();
+  }
+}
+
```
Comment (kasperg) [inline web/modules/custom/dbcdk_community/dbcdk_community.module]: Why is this included in the PR?

It does not seem related to the change at hand.

**Comment by kasperg** [comment]
Could we enable the Core Search module to support better content search?

File: web/config/sync/user.role.admin_library_employee.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+uuid: b598b954-c1e9-4a6f-a098-fafac21b0671
+langcode: en
+status: true
+dependencies: {  }
+id: admin_library_employee
+label: 'Library employee'
+weight: 5
+is_admin: null
+permissions:
+  - 'dbcdk community view reviews'
```
Comment (kasperg) [inline web/config/sync/user.role.admin_library_employee.yml]: Multiple unrelated files here as well.

File: web/modules/custom/dbcdk_community_moderation/dbcdk_community_moderation.module (hunk 1 above)
Comment (kallehauge) [inline web/modules/custom/dbcdk_community_moderation/dbcdk_community_moderation.module:22]: Hmm, I shouldn't have touched anything in the moderation module so that's just a rebase (or something) gone wrong. I'll remove the removal :smile:

**Comment by kallehauge** [comment]
Brilliant idea to take advantage of the `search` module! :+1: There is now added a search field instead of the title field that also searches in paragraphs. - There is no support for partial word searches though. I have tried to look at `search_api` but it is not yet ready for production sites. The newest alpha release fails multiple places (and I'm still not sure it it can do what we wish for or if it requires its own index-views like in D7, since I could barely install it).

File: web/modules/custom/dbcdk_community_content/dbcdk_community_content.info.yml [hunk 4]
```diff
@@ -6,3 +6,4 @@ version: 8.x-1.0
 core: 8.x
 dependencies:
   - dbcdk_community
+  - views
```
Comment (kasperg) [inline web/modules/custom/dbcdk_community_content/dbcdk_community_content.info.yml:10]: Do we depend on search here?

File: web/config/sync/search.page.node_search.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+uuid: 82431853-112f-46c2-a5bd-fd4f696ebc23
+langcode: en
+status: true
```
Comment (kasperg) [inline web/config/sync/search.page.node_search.yml:3]: Should the node search page be enabled? It is not used and we do not enable user search either.

Comment (kallehauge) [inline web/config/sync/search.page.node_search.yml:3]: It is not possible to disable the default search page (through the UI at least). So I disabled the User page - but I had to leave this one.

Comment (kasperg) [inline web/config/sync/search.page.node_search.yml:3]: :+1:
